src/components/Calculator.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './calculator.css';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Kindly remove this` eslint-disable` linters line, it always a good practice to fix linters manually instead of disable it

Reply:
This change has been done well